Formula/pyinvoke.rb

```diff
@@ -0,0 +1,38 @@
+class Pyinvoke < Formula
+  include Language::Python::Virtualenv
+
+  desc "Pythonic task management & command execution"
+  homepage "http://pyinvoke.org/"
+  url "https://github.com/pyinvoke/invoke/archive/0.13.0.tar.gz"
+  sha256 "4942aa901618f99afa70271d7b15fd018da0f174178fbca4048354c785e7736a"
+
+  head "https://github.com/pyinvoke/invoke.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    virtualenv_install_with_resources
```

Comment:
Yeh, not really your issue here just interested what @tdsmith thinks re: naming etc.